src/requirements/types.ts

```diff
@@ -154,7 +154,7 @@ export type SingleMenuRequirement = {
   readonly ongoing: DisplayableRequirementFulfillment[];
   readonly completed: DisplayableRequirementFulfillment[];
   readonly name: string;
-  readonly group: string;
+  readonly group: 'UNIVERSITY' | 'COLLEGE' | 'MAJOR' | 'MINOR';
```

Comment:
Do we still need to include 'UNIVERSITY' if we merged university and college requirements?